src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupClient.java

```diff
@@ -1314,4 +1314,108 @@ public ElastigroupGetInstanceStatusResponse getInstanceStatus(String instanceId)
         return retVal;
     }
 
+    public Elastigroup getBeanstalkConfig(String environmentId, String region) {
```

Comment:
re-arrange the functions as is in openApi document

Reply:
Done